BuildResidentialHPXML/measure.rb

```diff
@@ -2093,23 +2088,37 @@ def arguments(model)
     arg.setDefaultValue(4000)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('pv_system_2_inverter_efficiency', false)
-    arg.setDisplayName('PV System 2: Inverter Efficiency')
-    arg.setUnits('Frac')
-    arg.setDescription('Inverter efficiency of the second PV system.')
-    args << arg
-
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('pv_system_2_system_losses_fraction', false)
-    arg.setDisplayName('PV System 2: System Losses Fraction')
-    arg.setUnits('Frac')
-    arg.setDescription('System losses fraction of the second PV system.')
+    battery_location_choices = OpenStudio::StringVector.new
+    battery_location_choices << Constants.Auto
+    battery_location_choices << 'none'
+    battery_location_choices << HPXML::LocationLivingSpace
+    battery_location_choices << HPXML::LocationBasementConditioned
+    battery_location_choices << HPXML::LocationBasementUnconditioned
+    battery_location_choices << HPXML::LocationCrawlspaceVented
+    battery_location_choices << HPXML::LocationCrawlspaceUnvented
+    battery_location_choices << HPXML::LocationCrawlspaceConditioned
+    battery_location_choices << HPXML::LocationAtticVented
+    battery_location_choices << HPXML::LocationAtticUnvented
+    battery_location_choices << HPXML::LocationGarage
+    battery_location_choices << HPXML::LocationExteriorWall
+    battery_location_choices << HPXML::LocationRoofDeck
+    battery_location_choices << HPXML::LocationOutside
+    battery_location_choices << HPXML::LocationOtherHousingUnit
+    battery_location_choices << HPXML::LocationOtherHeatedSpace
+    battery_location_choices << HPXML::LocationOtherMultifamilyBufferSpace
+    battery_location_choices << HPXML::LocationOtherNonFreezingSpace
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('battery_location', battery_location_choices, true)
+    arg.setDisplayName('Battery: Location')
+    arg.setDescription('The space type for the lithium ion battery location.')
+    arg.setDefaultValue('none')
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeIntegerArgument('pv_system_2_num_units_served', true)
-    arg.setDisplayName('PV System 2: Number of Units Served')
-    arg.setDescription("Number of dwelling units served by second PV system. Must be 1 if #{HPXML::ResidentialTypeSFD}. Used to apportion PV generation to the unit.")
-    arg.setUnits('#')
-    arg.setDefaultValue(1)
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument('battery_capacity', true)
+    arg.setDisplayName('Battery: Nominal Capacity')
+    arg.setDescription('The nominal capacity of the lithium ion battery.')
+    arg.setUnits('kWh')
```

Comment:
A few decisions we still have to make:
- [ ] Ask for kWh, Ah, or allow either?
- [ ] Ask for power (W), voltage, or allow either?
- [ ] Ask for nominal capacity, usable capacity, or allow either?

We should come to an agreement with input from @rchintala13 and @nmerket.

Reply:
So for now we will store NominalCapacity (kWh) in an extension element?

We are now introducing RatedPowerOutput as an input? Either NominalCapacity OR RatedPowerOutput must be supplied?

Reply:
Comments mean right now, not future.

Reply:
How do we use RatedPowerOutput (W)?